### PR TITLE
Ensure that data file and rule file names are incorporated in the Console output

### PR DESCRIPTION
**Pull request by dchakrav-github**
*Issue*: #146 

*Description of changes:*
Ensure that data file and rule file names are incorporated in the Console output

```
$ cfn-guard validate -r rules/ -d data/
data/scenario-1.test.json Status = FAIL
PASS/SKIP rules
rules/simpler.guard/check_parameter_validity                     SKIP
rules/simpler.guard/check_ip_procotol_and_port_range_validity    SKIP
FAILED rules
rules/simpler.guard/check_resource_type_and_parameter            FAIL
---
data/scenario-2.test.json Status = FAIL
PASS/SKIP rules
rules/simpler.guard/check_resource_type_and_parameter            PASS
rules/simpler.guard/check_ip_procotol_and_port_range_validity    SKIP
FAILED rules
rules/simpler.guard/check_parameter_validity                     FAIL
---
data/scenario-6.test.json Status = PASS
PASS/SKIP rules
rules/simpler.guard/check_resource_type_and_parameter            PASS
rules/simpler.guard/check_parameter_validity                     PASS
rules/simpler.guard/check_ip_procotol_and_port_range_validity    PASS
---
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by priyap286** [comment]
We need to do something similar for the test command.

File: guard/src/commands/validate.rs [hunk 1]
```diff
@@ -65,7 +65,8 @@ or rules files.
         } else if app.is_present("last-modified") {
             last_modified
         } else {
-            regular_ordering
+            // making default alphabetical
```
Comment (priyap286) [inline guard/src/commands/validate.rs:68]: What did regular_ordering offer?

**Review by omkhegde — APPROVED** [review]

File: guard/src/commands/validate.rs [hunk 2]
```diff
@@ -65,7 +65,8 @@ or rules files.
...
+            alpabetical
```
Comment (gandhek) [inline guard/src/commands/validate.rs:69]: Can we simplify the if...else to always default to "alphabetical" unless "last-modified" is present? Line #63 seems like an unnecessary explicit check adding another branch.

**Review by gandhek — APPROVED** [review]
Minor comment in line, but overall LGTM

**Review by gandhek — APPROVED** [review]

**Pull request merged**